modules/Rock.cmake

```diff
@@ -211,13 +211,7 @@ macro(rock_standard_layout)
     if (IS_DIRECTORY ${PROJECT_SOURCE_DIR}/test)
         option(ROCK_TEST_ENABLED "set to OFF to disable the unit tests. Tests are automatically disabled if the boost unit test framework is not available" ON)
```

Comment:
I actually missed that one. You can't remove the conditional "enable if boost-unit-test is present" logic if ROCK_TEST_ENABLED is set to ON by default. It would break virtually all installs where boost-unit-test is not installed.

What about turning it to OFF, which is anyways what autoproj does by default ?